frontend/src/styles.module.css

```diff
@@ -55,3 +56,20 @@
   display: block;
   margin: auto;
 }
+
+.colorModeSwitch {
+  position: absolute;
+  right: 1rem;
+  bottom: 1rem;
+  color: var(--text-secondary);
+  fill: var(--text-secondary);
+  background: var(--primary);
+  border-radius: var(--border-radius);
+  cursor: pointer;
+  padding: 0.125rem;
+}
+
+.colorModeSwitch:hover,
+.colorModeSwitch:focus {
+  background: var(--primary-light);
+}
```

Comment:
If I see correctly, this styling is solely used by the ColorModeSwitch component, so it should be placed in a component specific `ColorModeSwitch.module.css` file instead. This file is solely for styles shared by multiple components.